controller/controller.go

```diff
@@ -166,35 +166,45 @@ func (c *Controller) DeleteFunc(obj interface{}) {
 		// handle the reconciled runtime object, we stop here.
 		return
 	} else if err != nil {
-		c.logger.Log("event", "delete", "function", "DeleteFunc", "level", "error", "message", "stop reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
+		c.logger.Log("level", "error", "message", "stop reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
 		return
 	}
 
 	ctx, err := resourceSet.InitCtx(context.Background(), obj)
 	if err != nil {
-		c.logger.Log("event", "delete", "function", "DeleteFunc", "level", "error", "message", "stop reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
+		c.logger.Log("level", "error", "message", "stop reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
 		return
 	}
 
+	{
+		meta, ok := loggermeta.FromContext(ctx)
+		if !ok {
+			meta = loggermeta.New()
+		}
+		meta.KeyVals["event"] = "delete"
+
+		ctx = loggermeta.NewContext(ctx, meta)
+	}
```

Comment:
I removed the `event` key from the logs and add it in general so we have it in all logs and not just the ones directly coming from the controller here.

Reply:
Just to understand. With this logging context "event" will be logged automatically?

Reply:
Exactly. We already do this with the `object` key which holds the k8s self link of the runtime object. See also https://github.com/giantswarm/operatorkit/blob/bd17983c3b092ef4f6a7db677a63614ed7adc3d6/controller/resource_set.go#L88.